www/js/diary/detail.js

```diff
@@ -47,6 +47,25 @@ angular.module('emission.main.diary.detail',['ui-leaflet', 'ng-walkthrough',
       $scope.$broadcast('invalidateSize');
   };
 
+  $scope.getIndividualSuggestion = function() {
+    var tripId = $stateParams.tripId;
+    console.log(tripId);
+    var uuID = $stateParams.uuID;
```

Comment:
are you filling this in somewhere? in general, the uuid does not need to be specified, it is implicit in the authentication used

Reply:
and you need to make sure that variables get put into the stateParams. stateParams is like scope, it is just a basic hash that you need to populate with your keys and values.